themes/default/content/blog/announcing-speaker-lineup-pulumiup-2023/index.md

```diff
@@ -0,0 +1,108 @@
+
+### Luke Hoban - TypeScript Co-Creator and CTO
+
+Luke is CTO at Pulumi. He has worked on developer tools his entire career. He started out on Visual Studio, C#, and .NET in the early 2000s, later joined the ECMAScript standards body as a representative of Microsoft and then became one of the co-founders of the TypeScript programming language.
+
+## Featured Partners
+
+Thank you to our amazing partners for supporting PulumiUP! Visit these booths for tech talks, demos and Q&A about your favorite cloud platforms and developer tools:
+
+* [AWS](https://aws.amazon.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - Solutions Architects will be on hand to demo the latest AWS and Pulumi capabilities and help you get the most from the AWS platform.
+* [Google Cloud](https://cloud.google.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - Customer Engineers will help you accelerate your cloud journey with helpful tips and Q&A.  
+* [Microsoft](http://azure.microsoft.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - Cloud Advocates will be available to show you how to kickstart your projects on Azure.
+* [Equinix](https://www.equinix.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - Stop by the Equinix booth to learn all about the new Equinix Pulumi provider.
+* [CircleCI](https://circleci.com/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - Stop by the CircleCI booth to learn best practices for adopting CI/CD in your organization.  
+* [New Relic](https://developer.newrelic.com/pulumi/get-started-pulumi/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - The New Relic team will demonstrate how to get started with Observability as Code with Pulumi and New Relic.
+* [Firefly](https://www.gofirefly.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - If you have existing cloud resources you can convert them to Pulumi code easily with Firefly. Stop by the Firefly booth for a demo.
+* [Nitric](https://nitric.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - The Nitric team will be on site to show you how to create cloud-agnostic infrastructure code from your application code.  
+* [Klotho](https://klo.dev/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - The Klotho team will show you how devops teams an application teams can easily collaborate to moderize and scale infrastructure without disruptions.
+* [Port](https://www.getport.io/?utm_source=Pulumi.com&utm_medium=Website&utm_campaign=PulumiUP) - Join the Port team to learn how to setup a self-service portal for your developers.
+
+Want to attend PulumiUP 2023? [Register now!](https://www.pulumi.com/pulumi-up/)
```

Comment:
```suggestion
### Want to attend PulumiUP 2023? [Register now!](https://www.pulumi.com/pulumi-up/)
```

Reply:
is this a heading? if not we shouldn't make it a heading; we can add styles if you want to make it more prominent.

Reply:
Oh, if you can add styles that would work! I just wanted to make it bigger and bolder.

> is this a heading? If not we shouldn't make it a heading; we can add styles if you want to make it more prominent.